examples/tests/error/struct_from_json_4.w

```diff
@@ -0,0 +1,29 @@
+// Note that this test has to be alone because it needs to compile successfully and fail at preflight.
+// If it is run with other tests, subsequent failures will be ignored in snapshot.
+
+struct Person {
+  name: str;
+  age: num;
+}
+
+struct Advisor extends Person {
+  id: str;
+}
+
+struct Student extends Person {
+  advisors: Set<Advisor>; // <== Using Set instead of Array
+}
+
+// Try adding two of the same adivsor
+let invalidAdvisorInArray = {
+  name: "cool",
+  age: 22,
+  advisors: [
+    {id: "advisor1", name: "Bob", age: 34},
+    {id: "advisor1", name: "Bob", age: 34},
+  ]
+};
+
+Student.fromJson(invalidAdvisorInArray);
+//              ^ ERROR: unable to parse Student:
+// - instance.advisors contains duplicate item
```

Comment:
It's debatable whether this is an error or just results in a single entry. When you add the same item to a set twice, you don't get an error.